ajax/libs/ice/package.json

```diff
@@ -8,6 +8,13 @@
         "*.js",
         "*.map"
       ]
+    },
+    {
+      "basePath": "lib/es5",
+      "files": [
+        "*.js",
+        "*.map"
```

Comment:
I create the pull request from github, I made the requested changes and made a new pull request with squash commit

https://github.com/cdnjs/cdnjs/pull/11684